test/performance/odp_sched_perf.c

```diff
@@ -665,16 +697,30 @@ static int create_queues(test_global_t *global)
 		}
 
 		for (j = 0; j < num_event; j++) {
-			buf = odp_buffer_alloc(pool);
+			if (test_options->pool_type == ODP_POOL_BUFFER) {
+				odp_buffer_t buf = odp_buffer_alloc(pool);
 
-			if (buf == ODP_BUFFER_INVALID) {
-				printf("Error: Alloc failed %u/%u\n", i, j);
-				return -1;
-			}
+				if (buf == ODP_BUFFER_INVALID) {
+					ODPH_ERR("Error: alloc failed %u/%u\n", i, j);
+					return -1;
+				}
 
-			if (odp_queue_enq(queue, odp_buffer_to_event(buf))) {
-				printf("Error: Enqueue failed %u/%u\n", i, j);
-				return -1;
+				if (odp_queue_enq(queue, odp_buffer_to_event(buf))) {
+					ODPH_ERR("Error: enqueue failed %u/%u\n", i, j);
+					return -1;
+				}
+			} else {
+				odp_packet_t pkt = odp_packet_alloc(pool, event_size);
+
+				if (pkt == ODP_PACKET_INVALID) {
+					ODPH_ERR("Error: alloc failed %u/%u\n", i, j);
+					return -1;
+				}
+
+				if (odp_queue_enq(queue, odp_packet_to_event(pkt))) {
+					ODPH_ERR("Error: enqueue failed %u/%u\n", i, j);
+					return -1;
+				}
```

Comment:
There could be single enqueue call (outside of if-else):
```
   ev = odp_packet_to_event(pkt);
}

if (odp_queue_enq(queue, ev)) {
...

```

Reply:
Fixed in V3.